### DISCUSSION TITLE
Safari折腾之路 | Goalonez Blog

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#127] [Announcements]
https://blog.goalonez.site/blog/Safari%E6%8A%98%E8%85%BE%E4%B9%8B%E8%B7%AF.html 

 万变不离其宗

**Comment by luxuxl** [comment]
楼主有没有尝试过 Via
- 单手操作：Via 底部的按钮可以设置长按功能，可以满足我的需求
- 快捷搜索：Via 搜索栏上有各个搜索引擎的按钮（可以自定义搜索引擎），点击即可切换
- 脚本管理器：安卓确定有内置，iOS 不确定（好久不用 iOS 了）

但对于我个人而言最喜欢的是书签同步，我通过 Via 的内置同步功能，将书签同步至 NutStore，然后 Chrome 安装 floccus 实现跨浏览器的书签同步（ 但是 floccus 有点小毛病，不是很完美）
